frontend/src/libs/packages/notification/notification.package.ts

```diff
@@ -0,0 +1,40 @@
+import { toast } from 'react-toastify';
+import { type ValueOf } from 'shared/build';
```

Comment:
Do not use direct import from shared
```suggestion
import { type ValueOf } from '~/libs/types/types.js';
```